lib/Bits.css

```diff
@@ -5762,17 +5907,13 @@ strong {
     position: -webkit-sticky !important;
     position: sticky !important; } }
 
-.u-no-print {
-  display: initial !important; }
-  @media print {
-    .u-no-print {
-      display: none !important; } }
+@media screen {
+  .u-print-only {
```

Comment:
Looks like some changes made in other PRs have not been built - I haven't touched this.

Reply:
Looks like you're right. I think building the CSS is a step that's definitely going to have to happen on deploy!